code-challenges.test.js

```diff
@@ -12,35 +12,99 @@
 
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
+// Inputs: array of Objects
+// Output: undefined
+// Parameters: variable object
+
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
+// describe("capitalizer", () => {
+//   it("returns an array with a sentence about each person with their name capitalized", () => {
+
+//   const people = [
+//     { name: "ford prefect", occupation: "a hitchhiker" },
+//     { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+//     { name: "arthur dent", occupation: "a radio employee" }
+//   ]
+//     expect(capitalizer()).toEqual('["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]')
+//      })
+//    })
+
+// const capitalizer = (obj) => {
+//   // takes in an object and iterate over each value
```

Comment:
This is definitely heading in the right direction.  One thing to consider would be to iterate over the array so that you can access each individual object.  Then you have access to the name and can use some string interpolation.